core/src/main/java/com/linecorp/armeria/server/AnnotatedServiceBindingBuilder.java

```diff
@@ -96,6 +106,16 @@ public AnnotatedServiceBindingBuilder responseConverter(
         return this;
     }
 
+    /**
+     * Adds the given {@link ResponseConverterFunction}s to this {@link AnnotatedServiceBindingBuilder}.
+     */
+    public AnnotatedServiceBindingBuilder responseConverters(
+            List<ResponseConverterFunction> responseConverterFunctions) {
```

Comment:
- `Iterable<? extends ResponseConverterFunction>`
- How about deprecating `responseConverter(ResponseConverterFunction)` and adding `responseConverters(ResponseConverterFunction...)`?